presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/DetermineJoinDistributionType.java

```diff
@@ -236,6 +252,25 @@ private JoinNode getSyntacticOrderJoin(JoinNode joinNode, Context context, JoinD
         return joinNode.withDistributionType(REPLICATED);
     }
 
+    private boolean isHBO(PlanNode planNode, Context context)
```

Comment:
1. confidenceLevel high does not necessarily mean it's hbo (e.g. could be table scan stats or some other thing with high confidence that gets added in the future), so the method name doesn't accurately reflect what this method is checking
2. That being said, we don't care whether the stats are from hbo. We just want to choose the stats with the highest confidence.